src/nupic/engine/RegionImplFactory.cpp

```diff
@@ -59,6 +59,12 @@ namespace nupic
   // Allows the user to add custom regions
   void RegionImplFactory::registerPyRegion(const std::string module, const std::string className)
   {
+    // Verify that no regions exist with the same className
+    for (auto pyr=pyRegions.begin(); pyr!=pyRegions.end(); pyr++)
+      if (pyr->second.find(className) != pyr->second.end())
+        NTA_THROW << "A pyRegion with name '" << className << "' already exists. " <<
```

Comment:
Please use braces even when you only have one line inside the if statement condition